README.md

```diff
@@ -164,6 +164,17 @@ future.
 - `email`: Allstar would send an email to the repository administrator(s).
 - `rpc`: Allstar would send an rpc to some organization-specific system.
 
+### **Action configuration**
+
+Two settings are available to configure the issue action:
+
+- `issueLabel` is available at the organization and repository level. Setting it
+  will override the default `allstar` label used by Allstar to identify its
+  issues.
+
+- `issueRepo` is available at the organization level. Setting it will force all
+  issues created in the organization to be created in the repository specified.
```

Comment:
i've been looking everywhere but can't figure out what the syntax is for this setting -- is it just the name of the repo (e.g. "allstar"), a URL to the issues database, something else?

Reply:
(i see the code answers the question below but it'd be nice if the README was clearer)